### ISSUE TITLE
Jenkins pkgdown build failing

### ISSUE THREAD

**Issue by wlandau** [ropensci/targets#422]
## Prework

* [x] Read and agree to the [code of conduct](https://ropensci.org/code-of-conduct/) and [contributing guidelines](https://github.com/ropensci/targets/blob/main/CONTRIBUTING.md).
* [x] Confirm that your issue is most likely a genuine bug in the `targets` package itself and not a user error, a known limitation, or a bug in another package that targets depends on. For miscellaneous troubleshooting, please post to the [discussions](https://github.com/ropensci/targets/discussions) instead.
* [x] If there is [already a relevant issue](https://github.com/ropensci/targets/issues), whether open or closed, comment on the existing thread instead of posting a new issue.
* [N/A] Post a [minimal reproducible example](https://www.tidyverse.org/help/) like [this one](https://github.com/ropensci/targets/issues/256#issuecomment-754229683) so the maintainer can troubleshoot the problems you identify. A reproducible example is:
    * [N/A] **Runnable**: post enough R code and data so any onlooker can create the error on their own computer.
    * [N/A] **Minimal**: reduce runtime wherever possible and remove complicated details that are irrelevant to the issue at hand.
    * [N/A] **Readable**: format your code according to the [tidyverse style guide](https://style.tidyverse.org/).

## Description

`pkgdown::build_site()` runs fine locally, but https://dev.ropensci.org/blue/organizations/jenkins/targets/detail/targets/257/pipeline/#log-145 amd https://dev.ropensci.org/blue/organizations/jenkins/targets/detail/targets/257/pipeline/#log-146 failed. @jeroen, do you have any troubleshooting advice?

**Comment by jeroen** [comment]
Is this some change in rlang? 

https://github.com/r-lib/rlang/commit/0731e27944c875ce02c3a9c79040ebc2fccba80d

**Comment by wlandau** [comment]
Thanks, seems to have worked locally using https://github.com/r-lib/rlang/commit/d68bf87feb48b666b0497d0455c7d64ef6af8195.

**Comment by wlandau** [comment]
So then do the Jenkins builds use development `rlang`?

**Comment by wlandau** [comment]
> Thanks, seems to have worked locally using https://github.com/r-lib/rlang/commit/d68bf87feb48b666b0497d0455c7d64ef6af8195.

To clarify, I am still getting the error on Jenkins and cannot reproduce locally.

**Comment by jeroen** [comment]
I think the version of rlang was cached.... I flushed the cache and it seems to work now. Sorry about this, seems we got a bad version of rlang at an unfortunate time.

> So then do the Jenkins builds use development rlang?

This happened because we get the dependencies from https://ropensci.r-universe.dev/ and because another ropensci package uses rlang in their remotes, it becomes part of the universe. Unfortunate in this case. I'll think about how to make this more robust.
